controllers/managedfusionoffering_controller.go

```diff
@@ -149,7 +151,7 @@ func (r *ManagedFusionOfferingReconciler) reconcileCatalogSource() error {
 		if err := r.own(r.catalogSource); err != nil {
 			return err
 		}
-		desiredCatalogSourceSpec := pluginGetDesiredCatalogSourceSpec(r)
+		desiredCatalogSourceSpec := pluginGetDesiredCatalogSourceSpec(r.managedFusionOffering)
```

Comment:
Not sure why you decided to stop sending the reconciler, please add it back
```suggestion
		desiredCatalogSourceSpec := pluginGetDesiredCatalogSourceSpec(r, r.managedFusionOffering)
```